_includes/header.html

```diff
@@ -19,6 +19,7 @@
               <li><a href="{% link index.html %}#get-started">Get started</a></li>
               <li><a href="{% link index.html %}#use-cases">Use Cases</a></li>
               <li><a href="{% link index.html %}#faq">FAQ</a></li>
+              <li><a href="https://doc.riot-os.org/">Documentation</a></li>
```

Comment:
If we add it, we should follow the convention for other external links that we use in the navbar already:
```suggestion
              <li><a href="https://doc.riot-os.org/" target="_blank">Documentation <i class="bi bi-box-arrow-up-right align-text-bottom"></i></a></li>
```